src/stories/OSCALControlProseByComponent.stories.js

```diff
@@ -0,0 +1,26 @@
+// Storybook docs for Replaced Prose with By-Component Parameter Value
```

Comment:
I think we can delete this file now that we've consolidated in `OSCALControlProse.stories.js`?